src/common/options.cc

```diff
@@ -1919,6 +1919,10 @@ std::vector<Option> get_global_options() {
     .set_default(8)
     .set_description(""),
 
+    Option("osd_skip_data_digest", Option::TYPE_BOOL, Option::LEVEL_ADVANCED)
```

Comment:
i don't think this is necessary with the `has_internal_csum()` method. @liewegas and @jdurgin , what do you think?

Reply:
Fine with me:-)

Reply:
Yeah i don't mind either way honestly!